lib/active_storage_validations/aspect_ratio_validator.rb

```diff
@@ -54,45 +53,45 @@ def validate_each(record, attribute, _value)
     private
 
 
-    def is_valid?(record, attribute, metadata)
+    def is_valid?(record, attribute, metadata, flat_options)

```

Comment:
This additional depencency is not needed for this method. You can calculate the flat_options in this method since you have the record already.

Reply:
True, I changed it.